demos/object_detection_demo_ssd_async/main.cpp

```diff
@@ -319,7 +319,6 @@ int main(int argc, char *argv[]) {
                 for (int i = 0; i < maxProposalCount; i++) {
                     float image_id = detections[i * objectSize + 0];
                     if (image_id < 0) {
-                        std::cout << "Only " << i << " proposals found" << std::endl;
```

Comment:
What's the purpose of this?

Reply:
This print is not really useful and buggy (If there are exactly `maxProposalCount` detections, the print will not be triggered)